### PR TITLE
Netcraft - Changed order of parameters and added unsecure

### PR DESCRIPTION
**Pull request by guykeller**
## Status
Ready

## Description
Changed the order of the parameters and added a new unsecure parameter.

## Does it break backward compatibility?
   - No

## Must have
- [ ] Code Review

### CODE REVIEW

File: Integrations/Netcraft/Netcraft.yml [hunk 1]
```diff
@@ -6,19 +6,23 @@ display: Netcraft
...
+  type: 9
+  required: true
+- display: The maximum number of entries (takedowns/notes) to return. Default is 100.
   name: limit
   defaultvalue: "100"
   type: 0
   required: false
-- display: Credentials
-  name: credentials
+- display: Use system proxy
```
Comment (Itay4) [inline Integrations/Netcraft/Netcraft.yml]:
```suggestion
- display: Use system proxy settings
```

File: Integrations/Netcraft/Netcraft.yml [hunk 2]
```diff
@@ -6,19 +6,23 @@ display: Netcraft
...
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Trust any certificate (unsecure)
```
Comment (Itay4) [inline Integrations/Netcraft/Netcraft.yml]:
```suggestion
- display: Trust any certificate (not secure)
```

File: Integrations/Netcraft/Netcraft.yml [hunk 3]
```diff
@@ -6,19 +6,23 @@ display: Netcraft
 category: Authentication
 description: An integration for Netcraft, allowing you to open and handle takedown requests.
 configuration:
-- display: Use system proxy
-  name: proxy
+- display: Credentials
```
Comment (Itay4) [inline Integrations/Netcraft/Netcraft.yml]:
```suggestion
- display: Username
```

Comment (guykeller) [reply Integrations/Netcraft/Netcraft.yml]: Why do you think we should use "Username" and not "Credentials"?

Comment (Itay4) [reply Integrations/Netcraft/Netcraft.yml]: have you ever seen a login which asked you for `Credentials` and not for `Username` ?

Comment (guykeller) [reply Integrations/Netcraft/Netcraft.yml]: But it's a username and password, not just username, wouldn't you agree?

Comment (Itay4) [reply Integrations/Netcraft/Netcraft.yml]: Password is there by default

Comment (guykeller) [reply Integrations/Netcraft/Netcraft.yml]: Fair enough, you're right

File: Integrations/Netcraft/Netcraft.yml [hunk 4]
```diff
@@ -6,19 +6,23 @@ display: Netcraft
...
   type: 0
   required: false
-- display: Credentials
-  name: credentials
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Trust any certificate (not secure)
```
Comment (Itay4) [inline Integrations/Netcraft/Netcraft.yml]: why is this of type 9 and required? did you test this?

**Comment by Itay4** [comment]
why is this unmockable ?

**Comment by guykeller** [comment]
Closing this PR as a new PR for Netcraft exists. Will open a new PR for the reformatting after the more recent PR is approved and merged.

**Pull request closed without merging**